wiki/Skinning_Tutorial/en.md

```diff
@@ -6,16 +6,16 @@
 
 ## Getting Started
 
-To get started, open the game client, select [Options](/wiki/Options), and click `Open osu! folder`. Once the `osu!/` folder is opened, go into the `Skins/` folder and create a new folder. You can name it whatever you want, but keep in mind that osu! will display this name in the options. Inside this folder, you can add any skinning elements listed in the skin element lists.
+To get started, open File Explorer. enter %localappdata% on the path bar, and press enter. Then, go into the `osu!/` folder, then `Skins/`. Right click, and create a new folder. You can also go into osu!, settings, and open the skins folder, then make the folder. You can name it whatever you want, but keep in mind that osu! will display this name on the dropdown of the skin select. Inside this folder, you can add any of the skinning elements listed in the [Skinning Elements Lists](/wiki/Skinning#skin-elements-lists).
```

Comment:
shouldn't assume where the game is installed, the gist of the old version is fine for finding the skins folder (language is oddly casual, though)